webpack.config.js

```diff
@@ -1,46 +1,59 @@
-const path = require('path');
-const CopyPlugin = require('copy-webpack-plugin');
+const path = require("path");
+const CopyPlugin = require("copy-webpack-plugin");
 
 module.exports = {
-  mode: 'development',
-  entry: './src/index.jsx',
-  devtool: 'eval-source-map',
+  mode: "development",
+  entry: "./src/index.jsx",
+  devtool: "eval-source-map",
   output: {
-    path: path.join(__dirname, 'dist'),
-    filename: 'bundle.js',
+    path: path.join(__dirname, "dist"),
+    filename: "bundle.js",
   },
   devServer: {
     historyApiFallback: true,
-    contentBase: path.join(__dirname, 'dist'),
+    contentBase: path.join(__dirname, "dist"),
     writeToDisk: true,
   },
   resolve: {
-    extensions: ['.js', '.jsx'],
+    extensions: [".js", ".jsx"],
   },
   module: {
     rules: [
       {
         test: /\.jsx?$/,
         exclude: /node_modules/,
         use: {
-          loader: 'babel-loader',
+          loader: "babel-loader",
           options: {
-            presets: ['@babel/preset-env', '@babel/preset-react'],
+            presets: ["@babel/preset-env", "@babel/preset-react"],
           },
         },
       },
       {
         test: /\.css$/,
-        use: ['style-loader', 'css-loader'],
+        use: ["style-loader", "css-loader"],
+      },
+      {
+        test: /\.s[ac]ss$/,
+        use: [
+          "style-loader",
+          "css-loader",
+          {
+            loader: "sass-loader",
+            options: {
+              additionalData: '@import "./src/variables.scss";',
+            },
+          },
```

Comment:
už jsem na to trochu pozapomněl že existuje možná trochu lepší způsob jak vložit třeba variables do každého souboru. Asi bych použil z https://webpack.js.org/loaders/sass-loader/#sassoptions
```
{
            loader: "sass-loader",
            options: {
              sassOptions: {
                includePaths: ["src/variables"],
              },
            },
          },
```